Code/Editor/ViewPane.cpp

```diff
@@ -179,14 +166,22 @@ CLayoutViewPane::CLayoutViewPane(QWidget* parent)
     toolbar->installEventFilter(&m_viewportTitleDlg);
     toolbar->setContextMenuPolicy(Qt::CustomContextMenu);
     connect(toolbar, &QWidget::customContextMenuRequested, &m_viewportTitleDlg, &QWidget::customContextMenuRequested);
-
     setContextMenuPolicy(Qt::NoContextMenu);
-
+    
     if (QToolButton* expansion = AzQtComponents::ToolBar::getToolBarExpansionButton(toolbar))
     {
         expansion->installEventFilter(m_expanderWatcher);
     }
 
+    AzQtComponents::BreadCrumbs* prefabsBreadcrumbs =
+        qobject_cast<AzQtComponents::BreadCrumbs*>(toolbar->findChild<QWidget*>("m_prefabFocusPath"));
+    QToolButton* backButton = qobject_cast<QToolButton*>(toolbar->findChild<QWidget*>("m_prefabFocusBackButton"));
```

Comment:
question: Is it worth adding an assert to check these are always found just in case something gets renamed or moved? Might be useful (and keep the actual null check too to avoid a crash)

Reply:
Yeah I was thinking about adding one last night, will do!